### DISCUSSION TITLE
在云服务部署之后，执行api_server.py报错No sentence-transformers model found with name BAAI/bge-m3. Creating a new one with mean pooling.

### DISCUSSION THREAD

**Discussion by Zackaxng** [zai-org/ChatGLM3#1289] [Q&A]
在云服务部署之后，执行api_server.py报错No sentence-transformers model found with name BAAI/bge-m3. Creating a new one with mean pooling.

**Comment by Neu070224** [comment]
这一行可以先注释掉，试一试！

**Comment by BLANKBLK** [comment]
需要在huggingface上拉取这个模块才行吧？
